packages/tinacms/src/auth/TinaCloudProvider.tsx

```diff
@@ -189,6 +198,9 @@ export const TinaCloudProvider = (
   }
 
   setupMedia()
+  if (cms.flags.get('tina-admin') === true) {
+    setupAdmin()
```

Comment:
I know we're doing the same thing with `setupMedia`, but should this be inside a `useMemo` hook?